src/status_im/ui/screens/wallet/send/views.cljs

```diff
@@ -73,10 +73,13 @@
       (i18n/label sign-label)
       [vector-icons/icon :icons/forward {:color :white}]]]))
 
+(defn- empty-to? [to]
+  (and (not (nil? to)) (not= to "")))
```

Comment:
`(and (not (nil? to)) (not= to ""))` same as `(not (empty? to))` ?

Reply:
Hm... almost. `(not (empty? to))` would also pass if `to` is say a vector. Still, I agree it's a better readibility and that's what we need here more.

Reply:
also isn't it a `not-empty-to?` function rather than `empty-to?`

Reply:
@goranjovic right, for empty vector results are not the same. But this function is not going to take vector as i understand. Otherwise, it doesn't make sense to compare it to empty string.

Reply:
Yeah, that makes sense. Good catch btw